fhir-persistence/src/test/java/com/ibm/fhir/persistence/search/test/AbstractPLSearchTest.java

```diff
@@ -131,7 +131,7 @@ protected void assertSearchDoesntReturnSavedResource(String searchParamName, Str
      * @throws Exception
      */
     protected boolean searchReturnsResource(String searchParamName, String queryValue, Resource expectedResource) throws Exception {
-        List<? extends Resource> resources = runQueryTest(expectedResource.getClass(), persistence, searchParamName, queryValue, Integer.MAX_VALUE);
+        List<? extends Resource> resources = runQueryTest(expectedResource.getClass(), searchParamName, queryValue, Integer.MAX_VALUE);
```

Comment:
This just jumped out at me... searchParamName vs searchParamCode >?

Reply:
done